agenda.md

```diff
@@ -0,0 +1,20 @@
+## Agenda topic best practices
```

Comment:
We might want to clarify we're just talking about TG1 agendas here.

Reply:
Clarified that it's for plenary